app/src/main/res/layout-sw600dp-port/question_player_fragment.xml

```diff
@@ -194,4 +195,4 @@
         android:layout_height="132dp" />
     </FrameLayout>
   </FrameLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
added